### ISSUE TITLE
Fill up space isnt transparent since v0.4.0 

### ISSUE THREAD

**Issue by Thxbault** [luisbocanegra/plasma-panel-colorizer#26]
Hello, 

Before updating to version 0.4.0 , the blank plasmoid used to fill up space were hidden, but after updating to v0.4.0, they keep the same color as the other widgets.

Here is a screen capture: 

![image](https://github.com/luisbocanegra/plasma-panel-colorizer/assets/6758181/e25e9115-e77a-469a-bd9d-f60c960bf8e5)

Not sure if i missed something, i'm still playing around with your app (and so far i like it) 

Im on EndeavourOS , with plasma 6.0.2 

Let me know if you need more infos

**Comment by Thxbault** [comment]
My bad, as always, fixed itself after a reboot

**Comment by luisbocanegra** [comment]
Yeah, I forgot to add a note about this, The reason why this happened is due to the new format in which some settings are saved (basically changed the separator to a "|" instead of newlines). 

The solution to this is either add again those excluded widgets manually or restore the default settings in the general tab, **it will delete the current settings** then you start creating your config again based on these new defaults.
